src/base/system.h

```diff
@@ -865,13 +865,11 @@ int net_addr_comp_noport(const NETADDR *a, const NETADDR *b);
  * @param addr Address to turn into a string.
  * @param string Buffer to fill with the string.
  * @param max_length Maximum size of the string.
- * @param add_port add port to string or not
+ * @param add_port Whether or not to add the port to the string.
```

Comment:
```suggestion
 * @param add_port Whether to add the port to the string.
```
"or not" in "whether or not" is usually redundant.

Doesn't block merging.